### ISSUE TITLE
Couldn't export operator custom_domain::_DCNv2

### ISSUE THREAD

**Issue by prabhuiitdhn** [onnx/onnx#3648]
I have a custom pytorch model and which includes "_DCNv2" I am trying to convert to onnx model. But it gives following error:

> RuntimeError: ONNX export failed: Couldn't export operator custom_domain::_DCNv2

If I use below code to add an operator to pytoch It is working and converting ONNX model

```
class _DCNv2(Function):
    @staticmethod
    def symbolic(g, input, offset, mask, weight, bias, stride, padding, dilation, deformable_groups):
        stride = _pair(stride)
        padding = _pair(padding)
        dilation = _pair(dilation)
        it is working
        return g.op("_DCNv2", input, offset, mask, weight, bias, stride_i=stride, padding_i=padding, dilation_i=dilation, deformable_groups_i=deformable_groups, )
```

But If I return with

```
return g.op("custom_domain::_DCNv2", input, offset, mask, weight, bias, stride_i=stride, padding_i=padding, dilation_i=dilation, deformable_groups_i=deformable_groups, )

or 

return g.op("ai.onnx.contrib::_DCNv2", input, offset, mask, weight, bias, stride_i=stride, padding_i=padding, dilation_i=dilation, deformable_groups_i=deformable_groups, )
```

It gives an error and not converting to onnx model

`RuntimeError: ONNX export failed: Couldn't export operator custom_domain::_DCNv2`

Any help would be highly appreciable. Thank you.

**Comment by prabhuiitdhn** [comment]
Hi: I also have tried to  register_custom_op_symbolic but still the same error

```
class _DCNv2(Function):
    @staticmethod
    def symbolic(g, input, offset, mask, weight, bias, stride, padding, dilation, deformable_groups):
        stride = _pair(stride)
        padding = _pair(padding)
        dilation = _pair(dilation)
        return g.op("ai.onnx.contrib::_DCNv2", input, offset, mask, weight, bias, stride_i=stride, padding_i=padding,
                    dilation_i=dilation, deformable_groups_i=deformable_groups, )

        register_custom_op_symbolic('::ai.onnx.contrib', _DCNv2, 9)
```

Can anyone please help me out?

**Comment by jcwchen** [comment]
Hi @prabhuiitdhn,
I don't have too context about the converter so I don't know why it worked previously but failed now...

However, I saw an update here: https://github.com/pytorch/pytorch/pull/39802
Did you add `custom_opsets={"your_domain_name": your_opset_version}` while using `torch.onnx.export` to convert a model with custom op? Thanks.

**Comment by prabhuiitdhn** [comment]
@jcwchen Can you please tell what should be the version of torch for this updates? 

I have torch==1.2.0 installed and when I am trying to do it It is coming as an error:

`TypeError: export() got an unexpected keyword argument 'custom_opsets'`

**Comment by jcwchen** [comment]
torch >= 1.6 should support this update. Could you please give it a try? Thanks

**Comment by prabhuiitdhn** [comment]
@jcwchen: I can't try with torch>=1.6 because project dependency is on torch==1.2.0. Is there any other way to add operator in onnx with torch version==1.2.0?

**Comment by prabhuiitdhn** [comment]
Hi @jcwchen thanks for your quick response, I have solved it with pytorch==1.5.0. Much appreciated. Thanks again.
+Others & -@jcwchen: Please follow [https://github.com/microsoft/onnxruntime/issues/8635](this link) to solve the issue. Ask me If I can help. I have struggled a lot so, I am here and may help you to fix this too. Thank you.
